ngmix/tests/test_admom.py

```diff
@@ -208,45 +207,130 @@ def test_admom_find_cen(g1_true, g2_true, snr):
     assert np.abs(dcol_mean)/dcol_err < 5, (dcol_mean/dcol_err)
 
 
-@pytest.mark.parametrize('pixel_scale', [0.25, 0.125])
-@pytest.mark.parametrize('fwhm', [2, 0.5])
-@pytest.mark.parametrize('image_size', [53])
-@pytest.mark.parametrize('mom_fwhm', [2, 2.5])
-def test_admom_comp_to_gaussmom_flux(image_size, fwhm, pixel_scale, mom_fwhm):
```

Comment:
Removing this test makes it hard to figure out if the flux normalization is different between the two versions.

Reply:
At minimum, a test comparing to a Gaussian fit would be clarifying.

Reply:
The new test compares to the true input flux.